src/modules/Lan.rb

```diff
@@ -909,7 +909,6 @@ def ProposeVirtualized
         bridge_name = LanItems.new_type_device("br")
         next unless connected_and_bridgeable?(bridge_name, current, config)
         LanItems.current = current
-        LanItems.add_current_device_to_routing
```

Comment:
Using the current name does not help here, so adapted to permit passing the name explicitly.